docs/configuration/node-config.md

```diff
@@ -13,17 +13,32 @@ A commented example is accessible here: [quickwit.yaml](https://github.com/quick
 
 ## Common configuration
 
-| Property | Description | Default value |
-| --- | --- | --- |
-| version | Config file version. 0.4 is the only available value. |  |
-| node_id | Node ID of the instance (searcher or indexer). It must be unique in your cluster. If not set, a random ID is generated at each boot. |  |
-| cluster_id | Unique Id for the cluster this node will be joining. Should be set to a unique name to ensure clusters do not accidentally merge together. | "quickwit-test-cluster" |
-| listen_address | The IP address or hostname that Quickwit service binds to for starting REST and GRPC server and connecting this node to other nodes. By default, Quickwit binds itself to 127.0.0.1 (localhost). This default is not valid when trying to form a cluster. | 127.0.0.1 |
-| rest_listen_port | The port which to listen for HTTP REST API. | 7280 |
-| peer_seeds | List of IP addresses used by gossip for bootstrapping new nodes joining a cluster. This list may contain the current node address, and it does not need to be exhaustive on every node. |  |
-| data_dir_path | Path to directory where data (tmp data, splits kept for caching purpose) is persisted. This is mostly used in indexing. | `./qwdata` |
-| metastore_uri | Metastore URI. Can be a local directory or `s3://my-bucket/indexes` or `postgres://username:password@localhost:5432/metastore`. [Learn more about the metastore configuration](metastore-config.md). | `{data_dir}/indexes` |
-| default_index_root_uri | Default index root URI that defines the location where index data (splits) is stored. The index uri is build following the the scheme: `{default_index_root_uri}/{index-id}` | `{data_dir}/indexes` |
+| Property | Description | Env variable | Default value |
+| --- | --- | --- | --- |
+| version | Config file version. 0.4 is the only available value. |  |  |
+| cluster_id | Unique Id for the cluster this node will be joining. Should be set to a unique name to ensure clusters do not accidentally merge together. | QW_CLUSTER_ID | "quickwit-test-cluster" |
```

Comment:
Isn't default cluster id is `quickwit-default-cluster`? 

https://github.com/quickwit-oss/quickwit/blob/5ac87253f791b4608916284e20e21a6b7dcd5538/quickwit/quickwit-config/src/quickwit_config/serialize.rs#L37